### ISSUE TITLE
Auditable.php line 184 - Indirect modification of overloaded property

### ISSUE THREAD

**Issue by goatatwork** [owen-it/laravel-auditing#105]
Hello,

I'm doing a straight forward deletion of a User model. The User model is using Auditable. Quick workflow:

1. Submit (via Vuejs) to `/api/users/10` where 10 is the ID of the user I am deleting. This step hits the `destroy()` method on my UsersApiController.

```
    public function destroy(User $user)
    {
        $user->delete();
    }
```

2. I receive the following error:

```
ErrorException in Auditable.php line 184:
Indirect modification of overloaded property App\User::$old has no effect
```

The user **is** deleted, but because of the exception, the Vuejs function does not continue on to do the rest of its work. After a page reload, I confirm that the user is, in fact, gone.

A couple of troubleshooting notes... If I find and delete the user via tinker, it works without error. If I use tinker to new up the UsersApiController, and then pass this user through the `destroy()` method, it works.

I have no idea what "Indirect modification of overloaded property" means, but am trying to remedy that lack of understanding right now.

**Comment by anteriovieira** [comment]
Hello @goatatwork , Thank you for reporting. Fix  [#094fabe](https://github.com/owen-it/laravel-auditing/commit/094fabe3a9bf85e7d405220fcff4d9d21604f6b4#diff-a8986ade675a771d625baff38f96baae)

Please, use new version V3.0.3

**Comment by goatatwork** [comment]
All good here. Thank you very very much. Also, thank you for a great package!
